jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -568,7 +568,7 @@ Path getJarArtifact() throws IOException {
           Xpp3Dom configuration = (Xpp3Dom) execution.getConfiguration();
 
           boolean skip = Boolean.valueOf(getChildValue(configuration, "skip").orElse("false"));
```

Comment:
nit: `Boolean.parseBoolean` returns a `boolean` and might be the better option here, vs `valueOf` which returns `Boolean` objects?